packages/expo-av/src/Audio/Sound.ts

```diff
@@ -15,7 +15,37 @@ import { PitchCorrectionQuality } from '../Audio';
 import ExponentAV from '../ExponentAV';
 import { throwIfAudioIsDisabled } from './AudioAvailability';
 
+export type AudioChannel = {
+  /**
+   * All samples for this specific Audio Channel in PCM Buffer format (-1 to 1).
+   */
+  frames: number[];
+};
+
+/**
+ * A single sample from an audio source. The sample contains all frames (PCM Buffer values) for each channel of the audio,
+ * so if the audio is _stereo_ (interleaved), there will be two channels, one for left and one for right audio.
+ */
+export type AudioSample = {
+  /**
+   * Data from each Channel in PCM Buffer format.
+   */
+  channels: AudioChannel[];
+  /**
+   * The timestamp of this sample, relative to the Audio Track's timeline in seconds.
+   */
+  timestamp: number;
+};
+
 type AudioInstance = number | HTMLMediaElement | null;
+type AudioSampleCallback = ((sample: AudioSample) => void) | null;
+
+declare let global: {
+  __EXAV_setOnAudioSampleReceivedCallback:
+    | ((key: number, callback: AudioSampleCallback) => void)
+    | undefined;
+};
```

Comment:
tbh I don't know, I carelessly copied that part from Marc because it worked,

but changing this to
```ts
declare global {
  const __EXAV_setOnAudioSampleReceivedCallback: () => void;
}
```
Causes calling `global.__EXAV_setOnAudioSampleReceivedCallback(...);` to throw
```
Property '__EXAV_setOnAudioSampleReceivedCallback' does not exist on type 'typeof globalThis'.
```
🤔 

---

But hopefully this works 😁 
```ts
declare global {
  interface Global {
    __EXAV_setOnAudioSampleReceivedCallback: () => void;
  }
}
```